### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by rofinn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ Manifest.toml
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
+/usr
```
Comment (iamed2) [inline .gitignore]: What writes to `./usr`? Usually such things go in `./deps/usr`

Comment (rofinn) [reply .gitignore]: The temporary FTPServer defaults to writing certs and data there.

Comment (omus) [reply .gitignore]: I think `./deps/usr` is more typically used.

Comment (rofinn) [reply .gitignore]: 👌

**Comment by rofinn** [comment]
I wasn't planning on it. The API is pretty much what's posted in the README.

**Comment by iamed2** [comment]
The README has barely anything. You should add API docs.

File: .gitignore (hunk 1 above)
Comment (omus) [reply .gitignore]: I think you forgot to update this `.gitignore` rule

File: src/FTPServer.jl [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        f(server)
+    finally
+        close(server)
+    end
+end
+
+hostname(server::Server) = "localhost"
+port(server::Server) = server.port
+username(server::Server) = server.username
+password(server::Server) = server.password
```
Comment (omus) [inline src/FTPServer.jl]: A `uri(server::Server) = "ftp://$(username(server)):$(password(server))@$(hostname(server)):$(port(server))"` would be useful.

**Comment by rofinn** [comment]
Created issues #2 and #3 to fix up remaining issues later.

File: src/FTPServer.jl [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+const pyopenssl_crypto = PyNULL()
+const pyopenssl_SSL = PyNULL()
+const pyftpdlib_authorizers = PyNULL()
+const pyftpdlib_handlers = PyNULL()
+const pyftpdlib_servers = PyNULL()
+
+# Defaults from pyftpdlib example
+const USER = "user"
+const PASSWD = "12345"
+const HOST="localhost"
```
Comment (iamed2) [inline src/FTPServer.jl]: Missing spaces around `=`

**Pull request merged**